script_facade/api/fhir.py

```diff
@@ -15,8 +16,15 @@ def medication_order():
     patient_lname = 'Skywalker'
     patient_dob = '1977-01-12'
 
-    bundle = rx_history_query(patient_fname=patient_fname, patient_lname=patient_lname, patient_dob=patient_dob)
-    return bundle
+    med_order_bundle = rx_history_query(patient_fname=patient_fname, patient_lname=patient_lname, patient_dob=patient_dob)
+
+    if current_app.config['RXNAV_LOOKUP_ENABLED']:
+        extended_med_order_bundle = add_rxnorm_coding(
```

Comment:
pythonic to receive an altered bundle rather than the side-effect, but you're not returning `extended_med_order_bundle`?

Reply:
nice catch, fixed!